code/game/objects/effects/mines.dm

```diff
@@ -63,27 +214,32 @@
 	name = "oxygen mine"
 	var/gas_amount = 360
 	var/gas_type = "o2"
+	disarm_product = /obj/item/deployablemine/gas
 
 /obj/effect/mine/gas/mineEffect(mob/victim)
 	atmos_spawn_air("[gas_type]=[gas_amount]")
 
 
 /obj/effect/mine/gas/plasma
-	name = "plasma mine"
+	name = "incendiary mine"
 	gas_type = "plasma"
+	disarm_product = /obj/item/deployablemine/plasma
 
 
 /obj/effect/mine/gas/n2o
-	name = "\improper N2O mine"
+	name = "knockout mine"
 	gas_type = "n2o"
-
+	disarm_product = /obj/item/deployablemine/sleepy
 
 /obj/effect/mine/sound
 	name = "honkblaster 1000"
 	var/sound = 'sound/items/bikehorn.ogg'
+	var/volume = 100
```

Comment:
No volume variable please, theres no need for this to ever exist and its only going to end up in gamer bussing in mines with volume of 1000000